docker/uwsgi/index.html

```diff
@@ -1,73 +1,93 @@
 <head>
-  <title>µRaiden demo</title>
-
+  <title style="margin-top: font-color: black">µRaiden demos</title>
+  <script type="text/javascript" src="//code.jquery.com/jquery-3.2.1.js"></script>
+  <script type="text/javascript" src="js/microraiden.js"></script>
```

Comment:
Why microraiden.js is included here? This page doesn't need it, it's for the paywall.